src/ethereum/EthereumVerifier.js

```diff
@@ -365,7 +365,7 @@ class EthereumVerifier {
       // Unique hash for epoch, secret contract address, and nonce
       const hash = cryptography.hash(abi.rawEncode(
           ['uint256', 'bytes32', 'uint256'],
-          [params.seed, nodeUtils.add0x(secretContractAddress), nonce]
+          [params.seed.toString(10), nodeUtils.add0x(secretContractAddress), nonce]
```

Comment:
Several questions:
1. Shouldn't the `uint256` type corresponding to it should be changed as well? 
2. Now that it is a dynamic type, shouldn't we use the [newly function](https://github.com/enigmampc/enigma-p2p/blob/master/src/common/cryptography.js#L46) for dynamic types? @elichai , what do you think?

Reply:
1. No, because the seed is indeed a number, and it will be encoded as a number. The link I provided above (https://github.com/ethereumjs/ethereumjs-abi/blob/8431eab7b3384e65e8126a4602520b78031666fb/lib/index.js#L67) illustrates how it converts from a string to a number.

2. I'm confused, there is no dynamic type here, the types that are specified are static and those that correspond to the variables that we are passing.  What dynamic types do you refer to?

Reply:
OK, my mistake, the conversion to string confused me..